WebKit WPE main/HEAD doesn't build using the master/HEAD of WPEBackend-FDO

... the same doesn't happen with WPEBackend-FDO 1.8.0

``` diff
--- /home/bot/toolchain/sysroots/armv7at2hf-neon-poky-linux-gnueabi/usr/lib/pkgconfig/wpe-1.0.pc	2021-03-05 05:36:02.854345912 +0000
+++ /home/bot/toolchain_1.9.1/sysroots/armv7at2hf-neon-poky-linux-gnueabi/usr/lib/pkgconfig/wpe-1.0.pc	2021-03-05 00:56:37.717953845 +0000
@@ -5,7 +5,7 @@
 
 Name: wpe-1.0
 Description: The wpe library
-Version: 1.9.1
+Version: 1.8.0
 Requires: xkbcommon
 Cflags: -I${includedir}/wpe-1.0
 Libs: -L${libdir} -lwpe-1.0
```

There are changes in the includes that could be the source of this errors. Here the differences between 1.8.1 and 1.9: https://paste.debian.net/1188147/. Probably related with f461fd4d306436bcefa0bdbf1821a191d7462c38.

``` diff
commit f461fd4d306436bcefa0bdbf1821a191d7462c38
Author: Adrian Perez de Castro <aperez@igalia.com>
Date:   Thu Nov 26 15:47:43 2020 +0200

    Simplify public headers
    
    Remove unneeded inclusions of libwpe headers, preferring forward
    declarations of the needed types and make all the inclusions use double
    quoted paths (to prefer local header versions, as opposed to installed
    ones).
```

The issue only affects the `Tools/wpe/backends` in WebKit so far. For example, you still can build WPE by disabling the tools (`-DENABLE_TOOLS=OFF`)

```cpp
../../Tools/wpe/backends/HeadlessViewBackend.cpp:85:22: error: ‘wpe_view_activity_state_visible’ was not declared in this scope
```

Full list of errors: https://paste.debian.net/1188149/

--- a/include/wpe/fdo-egl.h
+++ b/include/wpe/fdo-egl.h
@@ -30,6 +30,8 @@
 #ifndef __wpe_fdo_egl_h__
 #define __wpe_fdo_egl_h__
 
+#include <wpe/wpe.h>
+
 #define __WPE_FDO_EGL_H_INSIDE__
 
 #include "exported-buffer-shm.h"

--- a/include/wpe/fdo.h
+++ b/include/wpe/fdo.h
@@ -30,6 +30,8 @@
 #ifndef __wpe_fdo_h__
 #define __wpe_fdo_h__
 
+#include <wpe/wpe.h>
+
 #define __WPE_FDO_H_INSIDE__
 
 #include "version.h"

